import-dashboards.py

```diff
@@ -254,6 +257,18 @@ def set_home_dashboard(api_key):
     # cur.execute("REPLACE INTO preferences (id, org_id, user_id, version, home_dashboard_id, timezone, theme, created, updated) "
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
+def set_org_timezone(api_key):
+    r = requests.get('%s/api/org/preferences' % (HOST,), headers=grafana_headers(api_key))
+    print ' * Organization preferences: %r %r' % (r.status_code, r.content)
+    if r.status_code != 200:
```

Comment:
Please use `HTTPStatus.OK` from `http` lib.

Reply:
Looks like such lib has come up since the version 3.5

Reply:
I have replaced it by the OK constant from httplib